rpcs3/Emu/system_config.h

```diff
@@ -100,11 +100,13 @@ struct cfg_root : cfg::node
 	{
 		node_video(cfg::node* _this) : cfg::node(_this, "Video") {}
 
+		cfg::_enum<video_renderer> renderer{ this, "Renderer",
 #ifdef __APPLE__
-		cfg::_enum<video_renderer> renderer{ this, "Renderer", video_renderer::vulkan };
+											video_renderer::vulkan
```

Comment:
why don't you just write the whole member for each path?
two lines less and less confusing to read.
same below

Reply:
Oops

Reply:
Now I've made it similar to line 78

Reply:
still don't get why you don't just copy the whole line.
But I guess you're being influenced by existing code

Reply:
Should I change all three?

Reply:
sure why not.